include/exadg/solvers_and_preconditioners/multigrid/multigrid_preconditioner_base.cpp

```diff
@@ -984,6 +984,8 @@ MultigridPreconditionerBase<dim, Number>::do_initialize_transfer_operators(
   dealii::MGLevelObject<std::shared_ptr<dealii::MGConstrainedDoFs>> & constrained_dofs,
   unsigned int const                                                  dof_index)
 {
+  (void)constrained_dofs;
```

Comment:
Realizing the changes requested here https://github.com/exadg/exadg/pull/472#issuecomment-1576734029, this function can be simplified a lot I think. I believe the present PR should already do such changes and already adjust the interface of functions if some parameters are no longer needed.

Making such adjustments will help others in discussions such as https://github.com/exadg/exadg/issues/445 and related code changes / PRs.

Reply:
I'll do once we decide that this is the way to go.